packages/build/src/release.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { GithubRepo } from './github-repo';
+import release from './release';
+import { Octokit } from '@octokit/rest';
+import chai, { expect } from 'chai';
+import config from './config';
+import sinon from 'ts-sinon';
+
+chai.use(require('sinon-chai'));
+
+describe('Release', () => {
```

Comment:
Hmmm, I think probably not right now. I can remove this file for now.

Reply:
Why so? We should finish this.

Reply:
because of the structure of the release.ts all i'd be testing for is console.log. There are no other indicators of whether something is succeeded or failed. If you'd like to test for console.log, I guess we could do that.

Reply:
Is release just logging or doing something?